cli/popper/commands/cmd_init.py

```diff
@@ -30,8 +30,8 @@
 @click.option(
     '--infer-stages',
     help=('Infers the name of the stages from the existing bash scripts'
-          'on the folder while creating a new pipeline from the existing'
-          'folder.'),
+          ' on the folder while creating a new pipeline from the existing'
+          ' folder.'),
```

Comment:
maybe we could reword this to:

```
Infers the stages of a pipeline from bash script file names.
Used in conjunction with the --existing flag.
```

then user can read the help for `init` and for `--existing`